### PR TITLE
CFY-5224-Move-management-worker-packaging-to-omnibus

### PR DESCRIPTION
**Pull request by gilzellner**

### CODE REVIEW

File: packaging/mgmtworker/omnibus/package-scripts/mgmt-worker/postinst [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+function error_exit
+{
+  echo "${PROGNAME}: ${1:-"Unknown Error"}" 1>&2
+  exit 1
+}
+
+
+ln -sf $INSTALLER_DIR/embedded $INSTALLER_DIR/env || error_exit "Cannot link $INSTALLER_DIR/env"
+
+echo "Thank you for installing mgmt-worker!"
```
Comment (gilzellner) [inline packaging/mgmtworker/omnibus/package-scripts/mgmt-worker/postinst]: cloudify-mgmtworker

File: packaging/mgmtworker/omnibus/package-scripts/mgmt-worker/postrm [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+#
+# Perform necessary amqp-influxdb removal steps
+# after package is uninstalled.
+#
+
+echo "mgmtworker has been uninstalled!"
```
Comment (gilzellner) [inline packaging/mgmtworker/omnibus/package-scripts/mgmt-worker/postrm]: cloudify-mgmtworker
 Add a line note

File: packaging/mgmtworker/omnibus/package-scripts/mgmt-worker/preinst [hunk 3]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+#
+# before package is installed.
+#
+
+echo "You're about to install mgmt-worker!"
```
Comment (gilzellner) [inline packaging/mgmtworker/omnibus/package-scripts/mgmt-worker/preinst]: cloudify-mgmtworker
 Add a line note

**Comment by geokala** [comment]
Closed: As this PR is more than six months old without having been merged. Please reopen (and rebase if necessary) if it is still relevant.

**Pull request closed without merging**